README.md

```diff
@@ -43,6 +43,30 @@ You might be required to install using `sudo` privileges.
 
 Github Workflows are used. `./initialize.sh`is called, which attempts to build the Android and Desktop libraries and output .aar respectively .jar files. By default `./gradlew build` also runs `./gradlew test`.
 
+### Creating a Release
+
+This project follows [Semantic Versioning](https://semver.org) guidelines.
+
+To create a new release, it is possible to create one directly through the GitHub interface.
+
+However, it is instead recommended to create a tag locally:
+
+```bash
+git tag -a v0.1.0 "Annotation"
```

Comment:
This could be automated with `semantic-release` which will parse the staged commits and automatically tag the release